usr/tgtd.c

```diff
@@ -910,6 +912,58 @@ static int target_delete(const _ha_request *reqp,
 		return HA_CALLBACK_CONTINUE;
 	}
 
+	/*Unbind before delete*/
+	len = snprintf(cmd, sizeof(cmd),
+			"tgtadm --lld iscsi --mode target --op unbind --tid=%s"
+			" -I ALL", tid);
+	if (len >= sizeof(cmd)) {
+		set_err_msg(resp, TGT_ERR_TOO_LONG,
+			"tgt cmd unbind too long");
```

Comment:
Return proper error code, what is too long error ?

Reply:
Updated